contracts/vesting.token/include/vesting.token/vesting.token.hpp

```diff
@@ -0,0 +1,90 @@
+// vesting.token.hpp
+
+#pragma once
+
+#include <eosio/action.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/asset.hpp>
+#include <eosio/system.hpp>
+#include <eosio/singleton.hpp>
+
+namespace vestingtoken {
+    using eosio::action_wrapper;
+    using eosio::asset;
+    using eosio::name;
+    using std::string;
+    using eosio::check;
+
+    eosio::time_point sales_start_date;
```

Comment:
why time_point and not a time_point_sec?

this would Reduce the number of different unit types in the contract, simplifying things for developers